sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpSonarWayProfileTest.java

```diff
@@ -31,17 +31,24 @@
 public class CSharpSonarWayProfileTest {
 
   @Test
-  public void test() {
+  public void sonar_security_missing() {
     CSharpSonarWayProfile profileDef = new CSharpSonarWayProfile();
     BuiltInQualityProfilesDefinition.Context context = new BuiltInQualityProfilesDefinition.Context();
     CsRules.ruleKeys = new HashSet<>();
     profileDef.define(context);
     BuiltInQualityProfilesDefinition.BuiltInQualityProfile profile = context.profile("cs", "Sonar way");
     assertThat(profile.language()).isEqualTo(CSharpPlugin.LANGUAGE_KEY);
-    // FIXME : should be null when https://github.com/SonarSource/sonar-csharp/issues/1386 is fixed
-    assertThat(profile.rule(RuleKey.of(CSharpPlugin.REPOSITORY_KEY, "S3649"))).isNotNull();
```

Comment:
Just changing the `isNotNull` to `isNull` does not work because after we set `CsRules.ruleKeys` we need to call `profileDef.define` again to repopulate the quality profile rule list. It fails if you call it twice, hence I had to either recreate the context again, or split the test into two.